client/src/modules/dashboard/index.jsx

```diff
@@ -1,77 +1,185 @@
-import React, { Component } from 'react';
+import React, { Component, Fragment } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
 import _map from 'lodash/map';
 
+import Toolbar from 'common/components/Toolbar';
+import ToolbarItem from 'common/components/Toolbar/components/ToolbarItem';
```

Comment:
change exports so that we could do:
`import Toolbar, { ToolbarItem } from 'common/components/Toolbar' `